### DISCUSSION TITLE
TP4 - Ejercicio 1 - errores

### DISCUSSION THREAD

**Discussion by AgostinaApraiz** [INGCOM-UNRN-P1/cursada-2023#133] [Preguntas y respuestas] [answered]
Hola, que tal? Sigo teniendo problemas con el ejercicio, ya corregi lo que nos indicaron (cambiar int por char) pero me siguen apareciendo estos errores, y nose que esta mal.  
Muchas gracias

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/135774264/47b20daf-9a6e-45ab-a451-4e334bdb845c)

**Comment by martinvilu** [comment]
Mira que pasa si usas `%c` en los 3 `printf`

**Reply by AgostinaApraiz** [reply]
gracias, ahora aparece esto (lo guarde y abri denuevo la carpeta) siguen apareciendo estos errores.

![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/135774264/f89de9c8-8f4c-4639-9a43-2f08511f050f)

**Reply by martinvilu** [reply, marked as answer]
Falta una coma ahí.

Pero igual; decime que ves.

**Reply by AgostinaApraiz** [reply]
muchisimas gracias, ya funciona.

**Reply by martinvilu** [reply]
> Mira que pasa si usas `%c` en los 3 `printf`

Esto no es la respuesta para resolver el ejercicio; decime que ves... (y que es lo que esperas ver)

**Reply by AgostinaApraiz** [reply]
lo que espero ver es, en la primer linea la letra A, y en la segunda y tercer linea los lugares de memoria donde se encuentran los punteros.

lo que veo es esto
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/135774264/36179610-4af7-4cd3-b5a1-05809a2c1b97)

**Reply by martinvilu** [reply]
Yo no veo direcciones, solo veo la misma letra.

El `%p` es para punteros, pero el error en lo que me mostraste al principio esta en otro lado (aparte de la coma)

**Reply by AgostinaApraiz** [reply]
puede ser problema del visual entonces?

**Reply by martinvilu** [reply]
Visual, puede ser; un puntero se ve como un número hexadecimal mucho más grande. (maomeno `0x90AD394949399494`) 

La cosa es, recordá que hace el `*` del lado derecho de una expresión. (en especial adentro del `printf`)

**Reply by martinvilu** [reply]
Visual, en lo que se vé, no en lo que Visual Studio

![capisch](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/017c7dae-7688-44b2-b61a-92c43df657d1)

(recien me doy cuenta)
